canonical-webteam/global-nav
# Update with mongoose.ubuntu.com global nav JS

## Done

- Completely replaced the old `global.js` file with a new `global-nav.js` file written with ES6, using the styling and functionality found on mongoose.ubuntu.com.
- Note that at the moment the entirety of Vanilla is being injected into the document head - this will be fixed in https://github.com/ubuntudesign/web-squad/issues/748 in which the styling will be simplified and confined to the global nav itself.

## QA

- Pull code and run `yarn`
- Open `node_modules/vanilla-framework/scss/_patterns_slider.scss`, go to line 79 and replace with `width: calc(100% - (#{$thumb-size} / 2));` (it causes a build error with rollup-sass (see [issue](https://github.com/vanilla-framework/vanilla-framework/issues/1947)) - won't be an issue once https://github.com/ubuntudesign/web-squad/issues/748 is done)
- Run `yarn watch`
- Open `index.html`
- Check that the global nav looks and functions exactly like the one on mongoose.ubuntu.com
- Check copy, links and logos on large screens and mobile

## Issues

Fixes ubuntudesign/web-squad#749
